libs/blocks/chart/list.js

```diff
@@ -55,19 +54,80 @@ export const getListHtml = (data) => {
 
   return `
     <article>
-      <section class="title">${firstList.title}</section>
+      <section class="title">${list.title}</section>
       <section class="body">
         <${listType}>${listItems}</${listType}>
       </section>
     </article>
   `;
 };
 
+export const getCarouselHtml = (data) => {
+  const carouselItems = data.reduce((prev, list, idx) => (
+    `${prev}
+    <div class="carousel-item${idx === 0 ? ' active' : ''}"
+      role="group"
+      aria-roledescription="slide"
+      aria-label="${idx} of ${data.length}">
+      ${getListHtml(list)}
+    </div>
+    `
+  ), '');
+
+  return `
+    <section class="carousel">
+      <div class="controls">
+        <button type="button"
+          class="previous"
+          aria-controls="listChartCarousel-items"
+          aria-label="Previous Chart"></button>
+        <button type="button"
+          class="next"
+          aria-controls="listChartCarousel-items"
+          aria-label="Next Chart"></button>
+      </div>
+      <div id="listChartCarousel-items"
+        class="carousel-items"
+        aria-live="polite">
+        ${carouselItems}
+      </div>
+    </section>`;
+};
+
+const showCarouselItem = (element, index) => {
```

Comment:
Looks like you need a test for this one

Reply:
I was planning on adding this test after testing initList in MWPW-115223.